src/styles/blocks/header.scss

```diff
@@ -0,0 +1,16 @@
+.header {
+  display: flex;
+  box-sizing: border-box;
```

Comment:
it is a good practice to add `box-sizing: border-box;` on all elements using `*`